torch_frame/data/dataset.py

```diff
@@ -170,14 +172,18 @@ def __init__(
             # in-place sorting of col_names for each stype
             self._col_names_dict[stype].sort()
 
+        # Some multicategorical columns can be lists and thus
+        # do not need a separator.
         self.col_to_sep = canonicalize_col_to_pattern(
             col_to_sep,
-            self.col_names_dict.get(torch_frame.multicategorical, []),
+            self.col_names_dict.get(torch_frame.multicategorical),
+            all_inclusive=False,
```

Comment:
Can you clarify why we added `all_inclusive`? Seems to be set `False` for the main code.